bugsnag-android-core/src/main/java/com/bugsnag/android/NativeStackframe.kt

```diff
@@ -0,0 +1,52 @@
+package com.bugsnag.android
+
+import java.io.IOException
+
+/**
+ * Represents a single native stackframe
+ */
+class NativeStackframe internal constructor(
+
+    /**
+     * The name of the method that was being executed
+     */
+    var method: String?,
+
+    /**
+     * The location of the source file
+     */
+    var file: String?,
+
+    /**
+     * The line number within the source file this stackframe refers to
+     */
+    var lineNumber: Number?,
+
+    /**
+     * The address of the instruction where the event occurred.
+     */
+    var frameAddress: Long?,
+
+    /**
+     * The address of the function where the event occurred.
+     */
+    var symbolAddress: Long?,
+
+    /**
+     * The address of the library where the event occurred.
+     */
+    var loadAddress: Long?
+) : JsonStream.Streamable {
+
+    @Throws(IOException::class)
+    override fun toStream(writer: JsonStream) {
```

Comment:
It'd be good to create a unit test for this - `StackframeSerializationTest` would be a good starting point. It's a parameterized unit test that generates JSON, then loads a test fixture file from `test/resources/stackframe.json` to compare the expected vs observed JSON.

Reply:
It's already in https://github.com/bugsnag/bugsnag-android/pull/989/files#diff-6e669e8f7bb6e07399cd1971023f9ce8b119db6c1806f13359434fa63e919af3 and https://github.com/bugsnag/bugsnag-android/pull/989/files#diff-b8e696c2d2762d03cd3678387602472c30f89842f1d8fafe47c46adc42da936eR18

Reply:
Nice, beat me to it! :D